Xamarin.Forms.Platform.Android/Forms.cs

```diff
@@ -224,7 +224,8 @@ public override double ScalingFactor
 
 			protected override void Dispose(bool disposing)
 			{
```

Comment:
This needs to properly handle double disposal (i.e., it needs a 'disposed' flag). For an example, see https://github.com/xamarin/Xamarin.Forms/blob/ba6caad23ed7abdb1661cc61114c24eb3ae09da1/Xamarin.Forms.Platform.Android/AppCompat/MasterDetailContainer.cs#L113